synapse/python_dependencies.py

```diff
@@ -107,7 +107,7 @@
     # `systemd.journal.JournalHandler`, as is documented in
     # `contrib/systemd/log_config.yaml`.
     "systemd": ["systemd-python>=231"],
-    "url_preview": ["lxml>=3.5.0"],
+    "url_preview": ["lxml>=4.2.0"],
     "sentry": ["sentry-sdk>=0.7.2"],
     "opentracing": ["jaeger-client>=4.0.0", "opentracing>=2.2.0"],
     "jwt": ["pyjwt>=1.6.4"],
```

Comment:
These were bumped to be compatible with py 37 and up

Reply:
Have we run this one past the package maintainers?

Reply:
For completeness, judging by https://github.com/matrix-org/synapse/runs/4764560526?check_suite_focus=true#step:4:1379, it looks like the C-extension in lxml 3.5 wasn't compatible with CPython 3.8, but the C-extension in lxml 4.2.0 is compatible?

Reply:
yes that's my understanding-something about the C-headers changing?